api/v1alpha1/replicationdestination_types.go

```diff
@@ -104,13 +104,19 @@ type ReplicationDestinationRsyncSpec struct {
 	SSHUser *string `json:"sshUser,omitempty"`
 }
 
+// ReplicationDestinationRcloneSpec defines the field for rclone in replicationSource.
 type ReplicationDestinationRcloneSpec struct {
 	ReplicationDestinationVolumeOptions `json:",inline"`
-	// rcloneConfig is the name of a Secret that contains a valid Rclone
-	// configuration file.
-	RcloneConfig string `json:"rcloneConfig,omitempty"`
-	// path is the remote path to sync from.
-	Path string `json:"path,omitempty"`
+	//RcloneConfigSection is the section in rclone_config file to use for the current job.
+	RcloneConfigSection *string `json:"rcloneConfigSection,omitempty"`
+	//Direction is the direction of data movement.
+	Direction *string `json:"direction,omitempty"`
```

Comment:
Doesn't ReplicationSource vs ReplicationDestination determine the direction? (I may not understand what this parameter is for)

Reply:
When I started writing this I didn't see the direction in `ReplicationSourceSpec` or `ReplicationDestinationSpec`. I still don't see it referenced anywhere. Since it was specific for rclone I kept it under `ReplicationSourceRcloneSpec`

Reply:
My point was that if it's a ReplicationSource, we know that the direction is "source" (PV ==> remote) and if it's a ReplicationDestination, then it's remote ==> PV. Is there a case where a ReplicationSource would have a Direction that's not "source" (and similar for destination)?

Reply:
Ack. Logging issue for this.

Reply:
https://github.com/backube/scribe/issues/71